src/libs/utils.js

```diff
@@ -89,7 +89,7 @@ export const isOwner = hasAccessLevel('OWNER')
 export const workflowStatuses = ['Queued', 'Launching', 'Submitted', 'Running', 'Aborting', 'Succeeded', 'Failed', 'Aborted']
 
 export const log = (...args) => {
-  console.log.apply(null, args)
+  console.log.apply(null, args) // eslint-disable-line no-console
```

Comment:
I believe this is unused code-- please delete it.

Reply:
I believe this function is intended to help with debugging `_.flow` pipelines.

Reply:
How?

Reply:
If it's useful, please add documentation.

Reply:
To log an intermediate value in a pipeline, it's a shortcut to write:

```js
_.flow(
  step1,
  Utils.log,
  step2
)
```

instead of

```js
_.flow(
  step1,
  value => {
    console.log(value)
    return value
  },
  step2
)
```

Will add docs.

Reply:
Then again, I guess that's a route for console logs to go undetected by ESLint.

Reply:
Deleted in 794d78c66d224af885a6a2b51bb231af4c60d713.

Reply:
It provides a neat way to wrap a function to log something, like so:

![image](https://user-images.githubusercontent.com/5438223/162061147-ccb17607-75a6-486d-a771-1ab1df90a147.png)

I'd prefer to keep it.

Reply:
That seems like more of a quirk of the implementation than intended usage.

If we want something that wraps a function with a console log, I'd suggest we add something like:
```js
const withLog = (message, fn) => ...
```

Reply:
But that would still provide a route for console logs to go undetected by ESLint, when it seems like we always want to remove console logs before merging to dev.

Reply:
> That seems like more of a quirk of the implementation than intended usage.

Yea your `_.flow` example was a better one. 

> But that would still provide a route for console logs to go undetected by ESLint, when it seems like we always want to remove console logs before merging to dev.

I think the benefit outweighs the risk of that happening but I'm okay with letting it go.